packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -71,23 +57,22 @@ describe('Approve token', () => {
           cy.findByRole('button', {
             name: /Pay approval fee of/
           }).click({ scrollBehavior: false })
-          cy.confirmMetamaskPermissionToSpend()
+          cy.confirmMetamaskPermissionToSpend('1')
         })
-      cy.get('@transferButton').should('be.disabled')
-    })
 
-    it('should deposit successfully', () => {
-      // TODO: we don't have any indication in the UI that we are approving a token.
-      // We don't have a way to capture the finished approval state.
-      // Need better UX.
-      cy.log('Approving ERC-20...')
-      // eslint-disable-next-line
-      cy.wait(15000)
-      cy.confirmMetamaskTransaction().then(() => {
-        cy.findByText(
-          // PATCH : Find a proper fix later : `0.000000000001` will be rounded to 0 by our formatAmount function in tx cards
-          `Moving 0 ${ERC20TokenSymbol} to Arbitrum`
-        ).should('be.visible')
+      context('Deposit token', () => {
```

Comment:
Shouldn't this context be ideally outside of the `Approve token` context?

Reply:
Can't be outside because they run within the same test case, but I split it into two contexts: `Approve token` and `Deposit token`